src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -215,7 +258,8 @@ def process_workflows(job: WorkflowEventData) -> set[Workflow]:
         )
 
     with sentry_sdk.start_span(op="workflow_engine.process_workflows.evaluate_workflow_triggers"):
-        triggered_workflows = evaluate_workflow_triggers(workflows, job)
+        triggered_workflows = evaluate_workflow_triggers(workflows, event_data)
+        create_workflow_fire_histories(triggered_workflows, event_data)
```

Comment:
🤔 should `create_workflow_fire_histories` live in `evaluate_workflow_triggers`? that way if we evaluate a workflow and it's in a triggered state we would record that - this will only record if it's invoking process workflows